test/test_cromwell_workflow.sh

```diff
@@ -2,25 +2,26 @@
 
 set -e
 
-CROMWELL_USER=$1
-CROMWELL_PASSWORD=$2
-CROMWELL_URL=$3
-INPUTS_JSON=$4
-WDL_FILE=$5
-OPTIONS_FILE=$6
-DEPENDENCIES_JSON=$7
+CROMWELL_KEY_FILE=$1
+CROMWELL_URL=$2
+INPUTS_JSON=$3
+WDL_FILE=$4
+OPTIONS_FILE=$5
+DEPENDENCIES_JSON=$6
+COLLECTION=$7
 # Read list of dependency files into an array to pass to cromwell-tools
 DEPENDENCIES_LIST=$(cat ${DEPENDENCIES_JSON} | python3 -c "import json,sys;obj=json.load(sys.stdin);print(' '.join(obj.values()));")
```

Comment:
Could probably do this with jq too.

Reply:
@tbl3rd are you ok with this as is, or would you like to see a change?